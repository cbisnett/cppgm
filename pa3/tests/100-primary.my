0u
0u
0u
1u
2u
3u
error
error
error
error
0u
0u
0u
0u
0u
error
0u
0u
9223372036854775807u
error
error
error
error
error
eof
