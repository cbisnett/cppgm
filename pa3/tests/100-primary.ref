1
0
0
1
2
3
0u
1u
2u
3u
97
97u
97u
97
error
error
0
0
9223372036854775807
9223372036854775807
9223372036854775807
18446744073709551615u
18446744073709551615u
18446744073709551615u
eof
