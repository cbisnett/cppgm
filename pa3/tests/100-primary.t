true
false
0
1
0x2
03
0u
1u
0x2u
03u
'a'
u'a'
U'a'
L'a'
"foo"
3.2
ident
auto
9223372036854775807
9223372036854775807l
9223372036854775807ll
18446744073709551615u
18446744073709551615ul
18446744073709551615ull

