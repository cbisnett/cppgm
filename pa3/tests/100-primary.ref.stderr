ERROR: string-literal in controlling expression
ERROR: floating literal in controlling expression
